lib/util.go

```diff
@@ -34,3 +34,13 @@ func makeEndpointsList(headers []string) []string {
 
 	return endpoints
 }
+
+func quote(l []string) []string {
+	var quoted []string
+
+	for _, s := range l {
+		quoted = append(quoted, fmt.Sprintf("\"%s\"", s))
```

Comment:
Maybe use `"%q"`? :P Or, if you don't want to use it then use
```
`"%s"`
```
It's clearer, no escaping.